stdlib/public/core/TemporaryAllocation.swift

```diff
@@ -124,7 +124,7 @@ internal func _isStackAllocationSafe(byteCount: Int, alignment: Int) -> Bool {
 /// This function encapsulates the various calls to builtins required by
 /// `withUnsafeTemporaryAllocation()`.
 @_alwaysEmitIntoClient @_transparent
-internal func _withUnsafeTemporaryAllocation<T, R>(
+internal func _withUnsafeTemporaryAllocation<T: ~Copyable, R/*FIXME(NCG): ~Copyable*/>(
```

Comment:
?

Reply:
We can't yet do this due to rdar://124571365